app/assets/javascripts/reducers/columns/chillyBin.js

```diff
@@ -4,16 +4,27 @@ const initialState = {
   stories: [],
 };
 
+
+const orderByPosition = (items) => {
+  const orderedItems = [...items];
+
+  orderedItems.sort((a, b) => {
+    return parseFloat(a.position) - parseFloat(b.position)
```

Comment:
What do you think about creating a function outside `orderByPosition` called `comparePosition` with this so we don't re-create the function everytime? I know it's a small optimization but it's a really easy one so it's ok to do it now.